src/org/zaproxy/zap/extension/viewstate/HttpPanelViewStateView.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
```

Comment:
I guess it could be replaced with `Copyright 2016 The ZAP Development Team`, this is a new class even if similar to the core one.